src/main/java/org/opensearch/ad/indices/AnomalyDetectionIndices.java

```diff
@@ -786,8 +897,96 @@ public int getSchemaVersion(ADIndex index) {
      * @param index Index metadata
      * @return Whether the given index's mapping is up-to-date
      */
-    public Boolean isUpdated(ADIndex index) {
+    public Boolean isMappingUpdated(ADIndex index) {
```

Comment:
Seems this method is not being used now?

Reply:
yeah, removed